spec/features/users_task_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe '自身が作成したタスクの一覧' do
+  it do
+    user_a_auth_hash = generate_auth_hash
+    user_a = create_user_from_oauth_credential(generate_auth_hash)
+    user_b = create_user_from_oauth_credential(generate_auth_hash)
+
+    user_b_task = user_b.create_task('ユーザBのタスク').tap(&:save)
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `user_b_task`. Did you mean `user_a`?